x-pack/platform/plugins/shared/sample_data_ingest/public/services/utils.ts

```diff
@@ -8,6 +8,6 @@
 import { indexNamePrefix as sampleIndexNamePrefix, DatasetSampleType } from '../../common';
 
 export const isSampleIndex = (indexName: string) =>
-  Object.values(DatasetSampleType).some((sampleName) =>
-    indexName.startsWith(`${sampleIndexNamePrefix}-${sampleName.toLowerCase()}`)
+  Object.values(DatasetSampleType).some(
+    (sampleName) => indexName === `${sampleIndexNamePrefix}${sampleName.toLowerCase()}`
```

Comment:
could we use `getSampleDataIndexName()` here so that if we update the format there we don't need to keep it in sync here?